lib/Transport.js

```diff
@@ -30,7 +31,12 @@ Transport.prototype._action = function(method, url, data, callback) {
   // console.log(method, url, JSON.stringify(data, null, 2));
   var someRequest = request[method](url);
   this._attachAuthToRequest(someRequest);
-  someRequest.send(data).end(function(err, payload) {
+  if (method === "get") {
+    someRequest = someRequest.query(Perry.stringify(data));
+  } else {
+    someRequest = someRequest.send(data);
+  }
```

Comment:
This change gets around a bug/feature in `superagent` whereby the behaviour differs between serverside and the browser.